README.md

```diff
@@ -0,0 +1,9 @@
+2. Navigate to the repository directory:
+   ```
+   cd www2.naomimakkelie.nl
+   ```
+3. Install the required dependencies:
+   ```
+   bundle install
+   ```
+4. Serve the website locally:
```

Comment:
Just make a conventional README and explain how to add pages and manage the menu